tests/StructurizrPHP/Tests/Unit/Core/Documentation/AdrToolsImporter.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Structurizr for PHP.
+ *
+ * (c) Norbert Orzechowicz <norbert@orzechowicz.pl>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace StructurizrPHP\Tests\StructurizrPHP\Tests\Unit\Core\Documentation;
```

Comment:
I think you put it in wrong namespace, it should be `StructurizrPHP\AdrTools\`

Reply:
my bad, it should't by StructurizrPHP\StructurizrPHP\Core\Documentation ?

Reply:
No, it does not belongs to core, in java repo this is totally different module, `StructurizrPHP\AdrTools` is fine